File: fstp-core-bo/src/main/java/com/purefun/fstp/core/bo/SourceStockBO.java
package com.purefun.fstp.core.bo;

import java.io.Serializable;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import com.purefun.fstp.core.tool.UUID;

public class SourceStockBO  implements BaseBO {
	public String uuid = UUID.createUuid();					
	public long boid = 1501L;							
	public String destination = "fstp.ace.rds.source.stock";	
	public String secu_id = "";
	public String isin = "";
	public String record_update_time = "";
	public String secu_chinese_name = "";
	public String secu_english_name = "";
	public String secu_base_id = "";
	public String exch_type = "";
	public String secu_type = "";
	public String secu_sub_type = "";
	public String currency = "";
	public double bond_par_value = 0.0;
	public String not_list_stkqty = "";
	public String last_trade_date = "";
	public String list_date = "";
	public String product_set_id = "";
	public int buy_unit = 0;
	public int sell_unit = 0;
	public int trade_low_limit = 0;
	public int trade_high_limit = 0;
	public double pre_close_price = 0.0;
	public double tick_price = 0.0;
	public String price_limit_type = "";
	public double price_high_limit = 0.0;
	public double price_low_limit = 0.0;
	public double ex_right_ratio = 0.0;
	public double dividend_price = 0.0;
	public String financing_flag = "";
	public String margin_flag = "";
	public String secu_status = "";
	public String memo = "";
}

File: fstp-core-bo/python/com/purefun/fstp/core/bo/model/SourceStockBO.py
import uuid

class SourceStockBO(object):

    def __init__(self):
        self.secu_id = ''
        self.isin = ''
        self.record_update_time = ''
        self.secu_chinese_name = ''
        self.secu_english_name = ''
        self.secu_base_id = ''
        self.exch_type = ''
        self.secu_type = ''
        self.secu_sub_type = ''
        self.currency = ''
        self.bond_par_value = 0.0
        self.not_list_stkqty = ''
        self.last_trade_date = ''
        self.list_date = ''
        self.product_set_id = ''
        self.buy_unit = 0
        self.sell_unit = 0
        self.trade_low_limit = 0
        self.trade_high_limit = 0
        self.pre_close_price = 0.0
        self.tick_price = 0.0
        self.price_limit_type = ''
        self.price_high_limit = 0.0
        self.price_low_limit = 0.0
        self.ex_right_ratio = 0.0
        self.dividend_price = 0.0
        self.financing_flag = ''
        self.margin_flag = ''
        self.secu_status = ''
        self.memo = ''
        self.uuid = str(uuid.uuid1())
        self.boid = 1501
        self.destination = "fstp.ace.rds.source.stock"
